.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Thanks for contributing to RIOT !
+
+### Preliminary checks
+
+The RIOT community takes a lot of care to contributions quality.
+Thus before describing what your contribution is about, we would like you to
+verify the following checklist:
+- [ ] Have you read once the
```

Comment:
I would either remove "once" or put it at the end of the sentence (" at least once?")